06_01 GrayScale image Processing_GA1.obj : error LNK2019: _orimage_main 함수에서 참조되는 확인할 수 없는 외부 기호
06_01 GrayScale image Processing_GA1.obj : error LNK2019: _error_draw_circle 함수에서 참조되는 확인할 수 없는 외부 기호
C:\Users\User\source\repos\06일차\Debug\06일차.exe : fatal error LNK1120: 2개의 확인할 수 없는 외부 참조입니다.
